src/stream.rs

```diff
@@ -687,6 +776,18 @@ impl SendBuf {
         self.data.clear();
     }
 
+    /// Returns true if the send-side of the stream is complete.
+    ///
+    /// This happens when the stream's send final size is knwon, and the
```

Comment:
late review but typo `knwon`

Reply:
:+1: will fix in later PR.

Reply:
Fixed in https://github.com/cloudflare/quiche/commit/75d419fcf1446b9dda0b3a070e14900953841e87.